Add command to take screenshots of Shifts

jessarcher opened the pull request:
This PR adds a `screenshot:take` command that accepts a Shift ID and a path to save the image to.

Screenshots are currently taken at a fixed resolution of 1200x628 at 2x DPI, but a command-line option could be added for this.

In order for the command to be able to run without starting the dev server, I needed to inline the CSS and avatar image.

jasonmccreary commented:
@jessarcher made the tweaks we discussed the other day. Might need more as I didn't run it. But it's a start.

jessarcher commented:
@jasonmccreary Thanks! I fixed up a few things and also added image optimization. I opted to go with Spatie's `image` package (that was already installed as part of browsershot), however it does have some system requirements for optimization, which I have added to the README.

jasonmccreary commented:
Yeah, we'll see how easy these are to add to my system. GD is fine, but the other two I don't know. If so, we should just use the PNG Crush API so we don't add additional requirements to this project.
